The options to alter the vmsplit to enable more lowmem are hidden behind the
expert option. Make it more exposed for -ck users and make the help menu
more explicit about what each option means.

-ck

---
 arch/x86/Kconfig |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

Index: linux-4.1-ck1/arch/x86/Kconfig
===================================================================
--- linux-4.1-ck1.orig/arch/x86/Kconfig	2015-08-02 19:51:25.472833648 +1000
+++ linux-4.1-ck1/arch/x86/Kconfig	2015-08-02 19:51:25.471833630 +1000
@@ -1245,7 +1245,7 @@ config HIGHMEM64G
 endchoice
 
 choice
-	prompt "Memory split" if EXPERT
+	prompt "Memory split"
 	default VMSPLIT_3G
 	depends on X86_32
 	---help---
@@ -1265,17 +1265,17 @@ choice
 	  option alone!
 
 	config VMSPLIT_3G
-		bool "3G/1G user/kernel split"
+		bool "Default 896MB lowmem (3G/1G user/kernel split)"
 	config VMSPLIT_3G_OPT
 		depends on !X86_PAE
-		bool "3G/1G user/kernel split (for full 1G low memory)"
+		bool "1GB lowmem (3G/1G user/kernel split)"
 	config VMSPLIT_2G
-		bool "2G/2G user/kernel split"
+		bool "2GB lowmem (2G/2G user/kernel split)"
 	config VMSPLIT_2G_OPT
 		depends on !X86_PAE
-		bool "2G/2G user/kernel split (for full 2G low memory)"
+		bool "2GB lowmem (2G/2G user/kernel split)"
 	config VMSPLIT_1G
-		bool "1G/3G user/kernel split"
+		bool "3GB lowmem (1G/3G user/kernel split)"
 endchoice
 
 config PAGE_OFFSET
